pysat/tests/test_utils_io.py

```diff
@@ -230,6 +230,8 @@ def test_inst_write_and_read_netcdf(self):
                                 repr(netcdf_inst.meta[var, attr])))
 
         # Remove the local instrument
+        if hasattr(netcdf_inst.data, "close"):
+            netcdf_inst.data.close()
```

Comment:
```suggestion
```
Since it didn't work. :(

Looks good otherwise!

Reply:
we can probably drop the close statements (Lines 226-228) since we have the try/except on the test that breaks